UofT-Frosh-Orientation/orientation-website
# 171 create stripe routes and component

This PR expanded a bit as I was working on it. It contains the code to connect the registration frontend to the backend, handle the payments, and allow users to login (but not sign up). You will need to update your .env file to test this, so please message me for details.

To test:
1. Create an account for yourself using the api
2. Login to that account
3. Go to the registration page
4. Use the test cards from (stripe)[https://stripe.com/docs/testing] to test if the payment works
5. Upon a successful payment you will be redirected back to the orientation site and it will say on the orientation site that an error occurred - that is the expected behaviour currently, as there is a separate ticket to fix that page.
